fastlane/lib/fastlane/environment_printer.rb

```diff
@@ -231,11 +231,8 @@ def self.print_fastlane_files
     # Make sure to ask the user first, as some people don't
     # use a clipboard manager, so they might lose something important
     def self.copy_to_clipboard(string)
-      require 'tempfile'
-      Tempfile.create('environment_printer') do |tmp_file|
-        File.write(tmp_file, string)
-        `cat '#{tmp_file.path}' | pbcopy`
-      end
+      require 'open3'
+      Open3.popen3('pbcopy') { |input, _, _| input << string }
```

Comment:
Oh interesting, this will also work for special characters and new lines, right?